doc/source/whatsnew/v1.3.0.rst

```diff
@@ -75,8 +75,10 @@ Performance improvements
 Bug fixes
 ~~~~~~~~~
 
--
--
+I/O
```

Comment:
there is an I/O section below

Reply:
must have missed that. Will move it there